spectral_cubes/emission_search_cubes.md

```diff
@@ -0,0 +1,1101 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.18.1
+kernelspec:
+  display_name: python3
+  language: python
+  name: python3
+---
+
+# Beyond metadata: analytical data search in the cloud with JWST spectral cubes
+
++++
+
+## Learning goals
+
+In this notebook, we'll learn about how to answer data-intensive archival search queries in astronomical data archives like [MAST](https://archive.stsci.edu/). We'll have to think about:
+* when to parallelize our data processing,
+* the order of operations for performing cone searches on large numbers of targets,
+* when to load data into memory versus downloading to storage,
+* the advantages and constraints of a cloud science platform,
+* and when to load data from cloud storage instead of on-premise storage.
+
+## Introduction
+
+As you read through a recent paper by [Assani et al. 2025](https://arxiv.org/pdf/2504.02136), imagine that you find yourself consumed by a single thought:
+
+> I want to find more JWST spectral image cubes containing Fe II emission from jets launched by young stellar objects (YSOs).
+
+You may now be experiencing some despair. "This observation contains a JWST spectral cube" is certainly something readily available from MAST's metadata tables, but "this file contains Fe II transitions" is certainly not. So instead, you need to dig beyond the metadata and into the data of thousands of files. That may sound difficult, but it can be done!
```

Comment:
Please make the language more factual and less story book like. Do not assume the user is in despair and don't know how to approach a science question, etc.